.gitmodules

```diff
@@ -16,9 +16,6 @@
 [submodule "externals/spotless"]
 	path = externals/spotless
 	url = https://github.com/RobotLocomotion/spotless-pod.git
-[submodule "externals/director"]
-	path = externals/director
-	url = https://github.com/RobotLocomotion/director.git
```

Comment:
Sorry, my mistake. This submodule should be there. I've added it back to .gitmodules. (Originally, I was manually git pull the RobotLocomotion/director with the same SHA used by RobotLocomotion/drake master).